Sileo/Sileo/fr.lproj/Localizable.strings

```diff
@@ -96,46 +96,65 @@
 
 "Untitled_Form" = "Formulaire sans-titre";
 "Form_Submitted.Default_Title" = "Formulare soumis";
-"Form_Submitted.Default_Body" = "La soumission du formulaire à succédé.";
+"Form_Submitted.Default_Body" = "Le formulaire a été soumis avec succès";
 
 "Web_Depiction_Deprecated_Warning" = "Les représentations utilisant les Web View sont obsolets. Le gérant de la source doit se mettre à jour en utilisant les représantations natives.";
 
 "Package_Details_Tab" = "Détails";
 "Package_Changelog_Tab" = "Changements";
 "Package_No_Description_Available" = "Aucune description n'est disponible pour ce paquet.";
-"Package_Changelogs_Unavailable" = "Les changements ne sont pas disponibles pour ce paquet.";
+"Package_Changelogs_Unavailable" = "Pas de journal de changement disponible";
 
 "Installed_Package_Header" = "Paquet installé";
 "Version" = "Version";
 "Show_Package_Contents_Button" = "Dévoiler les contenus du paquet";
 
+"Licenses_Page_Title" = "Licenses";
+
 "Settings_Payment_Provider_Heading" = "Fournisseurs de paiement";
 "Settings_Translations_Heading" = "Traductions";
 
 "Settings_Payment_Provider.Purchases_Heading" = "Achats";
 "Settings_Payment_Provider.No_Purchases" = "Aucun achat";
 
+"Settings" = "Paramètres";
+"Settings_Page_Title" = "Paramètres";
+"Dark_Mode_Toggle" = "Mode Sombre";
+"Theme" = "Thème";
+"Tint_Color" = "Couleur de la teinte";
```

Comment:
Maybe "Couleur d'accentuation" would be better

Reply:
Yeah checked how mac os translated this in french, while I don't think it's better for the user better stick with the "official" translation